### ISSUE TITLE
Quest "Lights in the dark" unlimited Vial of Purifying Spirits

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#208]
```
This was reported here:
http://andors.techby2guys.com/viewtopic.php?f=3&t=863

During the quest "Lights in the dark" at the point where you get the "Vial of 
Purifying Spirits" from "Throdna". If you keep talking to him over and over he 
keeps giving you more "Vial of Purifying Spirit" each time over and over.

I wouldn't call it a big bug but its not logical to the flow of the game.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 9 Jul 2011 at 6:04

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. 

This is now fixed in r122.
```

Original comment by `oskar.wi...@gmail.com` on 13 Jul 2011 at 8:24
- Changed state: **Ready**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
